app/src/main/java/com/github/steroidteam/todolist/todo/Task.java

```diff
@@ -0,0 +1,64 @@
+package com.github.steroidteam.todolist.todo;
+
+import java.io.Serializable;
+
+/**
+ * A class that represents a task described by a title and a body.
+ */
+public class Task implements Serializable{
+
+    private String body;
+    private final String id;
```

Comment:
I would go with an (unsigned?) int for the ID.

Reply:
I go for an Integer, so we can create a task with null id:  not already in the database.